HidingNavigationBarSample/HidingNavigationBarSample/HidingNavExtensionViewController.swift

```diff
@@ -54,9 +54,8 @@ class HidingNavExtensionViewController: UIViewController, UITableViewDataSource,
 	// MARK: UITableViewDelegate
 	
 	func scrollViewShouldScrollToTop(_ scrollView: UIScrollView) -> Bool {
-		hidingNavBarManager?.shouldScrollToTop()
-		
-		return true
```

Comment:
We can use nil coalescing operator here.

return hidingNavBarManager?.shouldScrollToTop() ?? true

Reply:
@anirudhramanan updated

Reply:
Also need to mention this change in notes for new pod version

Reply:
yes